tests/test_freetextsearch.py

```diff
@@ -153,54 +1466,50 @@ def testBestHitSearchFieldsTarget(self):
         
         self.assertEqual(len(json_response['data']),6)
         
-        braf_data = json_response['data'][0];
-        self.assertEqual( braf_data['highlight'], '')
+        braf_data = json_response['data'][0]
```

Comment:
I would like them disabled for this case. @emepyc - what do you think? Or if not disabled - at least having an option to disable would be great.